### ISSUE TITLE
ThomasHaile: Week 1, Databases

### ISSUE THREAD

**Issue by thomashaile** [HackYourFutureBelgium/class-8#369]
Databases - SQL Lite3 [week-1](https://github.com/HackYourFutureBelgium/databases/tree/master/week-1)
---

#### Lesson Plan 
***Isolate***
Check out this week's [exercises here](https://github.com/thomashaile/databases/blob/master/isolate/week-1/EXERCISES.md). You can study them in the [SQLite Tutorial Sandbox](https://www.sqlitetutorial.net/tryit/).

***Integrate***
You will be learning to use sqlite3, a module that allows you to query your SQLite database file.
 - [ ] CLI Query Scripts

### Name of Project: restful-chinook

- [repository](github.com/user-name/repo-name)
- [live demo](https://user-name.github.io/repo-name)
- [ ]  [step 1](link to branch for version 1)
- [ ]  [step 2](link to branch for version 2)
- [ ]  [step 3](link to branch for version 3)
- [ ]  ...

### Exercises:

**Suggested Study**

***SQL***
 - [ ] sqlteaching
 - [ ] SQL Murder Mystery

***Tooling***
 - [ ] TablePlus is a modern, native, and friendly GUI tool for several relational databases: SQLite, 
     MySQL, PostgreSQL & more.
 - [ ] DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite.
 - [ ] SQLite Viewer is an online SQLite editor.
 - [ ] SQLite Editor by SQLite Tutorial is also an online SQLite editor, specifically for the Chinook sample database.
 - [ ] vscode-sqlite is a VSCode extension to explore and query SQLite databases.

***SQLite & Node.js***
 - [ ] SQLite Node.js Guide
 - [ ] node-sqlite3 has good docs
 - [ ] developerhowto tutorial article
 - [ ] PetsApp: video tutorial (SQLite starts at #7), starter code & finished to study

> [Follow this link](https://github.com/HackYourFutureBelgium/class-6/issues/59) to study a well-done homework issue.

**Comment by samicey** [comment]
Hello @thomashaile your homeworks are empty. There are no added codes in your controllers.

Please check it and get back to me.

**Comment by thomashaile** [comment]
No, i put it all branch by branch you can check it all. I left the master
empty for later use. And i put link for my client side. Thanks

**Comment by samicey** [comment]
Ok  great @thomashaile . Next time, you can always create a branch called **develop** where you will merge all your feature branches and test if the work well together.

By doing that you will stay have your master branch

**Comment by thomashaile** [comment]
Ok, thanks i will do that
